indico/modules/events/abstracts/operations.py

```diff
@@ -62,3 +62,28 @@ def delete_abstract(abstract, delete_contrib=False):
     logger.info('Abstract %s deleted by %s', abstract, session.user)
     abstract.event_new.log(EventLogRealm.management, EventLogKind.negative, 'Abstracts',
                            'Abstract "{}" has been deleted'.format(abstract.title), session.user)
+
+
+def schedule_cfa(event, data):
```

Comment:
I would still leave `start_dt, end_dt, modification_end_dt` instead of `data`. Otherwise it's quite cryptic to use from the shell.